### PR TITLE
Introduce PEP8 checks and clean the code

### PR DESCRIPTION
**Pull request by fivaldi**

### CODE REVIEW

File: pyvocz/views.py [hunk 1]
```diff
@@ -320,7 +319,7 @@ def event(series_slug, date_slug):
                                 date_slug=proper_date_slug))
 
     github_link = "https://github.com/pyvec/pyvo-data/blob/master/{filepath}" \
-            .format(filepath=event._source)
+        .format(filepath=event._source)
```
Comment (encukou) [inline pyvocz/views.py]: According to PEP 8, the [preferred way of wrapping long lines](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) is parentheses, not backslash. Please use that. (And teach it to the tool, if possible.)

File: .travis.yml [hunk 2]
```diff
@@ -5,9 +5,11 @@ python:
...
 install:
   - pip install -e.
-  - pip install pytest pytest-flask lxml requests cssselect
+  - pip install pytest pytest-flask lxml requests cssselect flake8
   - git clone --depth 1 https://github.com/pyvec/pyvo-data pyvocz/pyvo-data
 
-script: py.test test_pyvocz
+script:
+  - py.test test_pyvocz
+  - flake8 --max-line-length=100
```
Comment (encukou) [inline .travis.yml]: If we want to follow PEP 8, let alone check it automatically, let's not add exceptions. Wrap lines at 79 characters (and docstrings/comments at 72). (I actually consider this more important than a lot of the other rules – it allows me to put several windows side-by-side.)

File: pyvocz/views.py [hunk 3]
```diff
@@ -339,7 +338,7 @@ def event_qrcode(series_slug, date_slug):
         qr_byte_io = BytesIO()
         qr_img.save(qr_byte_io, 'PNG')
         qr_byte_io.seek(0)
-        cache.set(url, qr_byte_io, timeout=5*60)
+        cache.set(url, qr_byte_io, timeout=5 * 60)
```
Comment (encukou) [inline pyvocz/views.py]: I couldn't find the rationale for this change in [PEP 8](https://www.python.org/dev/peps/pep-0008).

File: setup.py [hunk 4]
```diff
@@ -17,18 +17,21 @@ def run_tests(self):
...
 def data_file_names():
     for top in 'pyvocz/static', 'pyvocz/templates':
         for dirname, dirs, files in os.walk(top):
             for filename in files:
                 fullname = os.path.join(dirname, filename)
                 if filename not in dirs and fullname.endswith((
                         '.png', '.svg', '.jpg', '.css', '.html',
-                        )):
+                )):
                     yield os.path.relpath(fullname, 'pyvocz')
```
Comment (encukou) [inline setup.py]: I believe this complied to [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation), no need to change it.

File: .travis.yml [hunk 5]
```diff
@@ -5,9 +5,11 @@ python:
...
 install:
   - pip install -e.
-  - pip install pytest pytest-flask lxml requests cssselect
+  - pip install pytest pytest-flask lxml requests cssselect flake8
   - git clone --depth 1 https://github.com/pyvec/pyvo-data pyvocz/pyvo-data
 
-script: py.test test_pyvocz
+script:
+  - py.test test_pyvocz
+  - flake8 --max-line-length=100 --exclude=pyvocz/pyvo-data/
```
Comment (encukou) [inline .travis.yml]: If we want to follow PEP 8, let alone check it automatically, let's not add exceptions. Wrap lines at 79 characters (and docstrings/comments at 72). (I actually consider this more important than a lot of the other rules – it allows me to put several windows side-by-side.)

File: setup.py (hunk 4 above)
Comment (fivaldi) [reply setup.py]: Still the indentation is visually more understandable (on first sight) than the "un-indentation".

Compare:
```
)):
    yield os.path.relpath(fullname, 'pyvocz')
```

To:
```
        )):
yield os.path.relpath(fullname, 'pyvocz')
```

File: pyvocz/views.py (hunk 3 above)
Comment (fivaldi) [reply pyvocz/views.py]: Still it's visually readable.

File: .travis.yml (hunk 5 above)
Comment (fivaldi) [reply .travis.yml]: OK, changed.

File: pyvocz/views.py (hunk 3 above)
Comment (encukou) [reply pyvocz/views.py]: This is not that much about this particular case, but about a tool forcing something that's not actually in PEP8.

**Comment by fivaldi** [comment]
@encukou What could be the reason of the failing test, please?
See: https://travis-ci.org/pyvec/pyvo.cz/builds/352775413#L622

File: setup.py (hunk 4 above)
Comment (encukou) [reply setup.py]: I disagree.

But again, I don't really care about this case. What I don't want is a checker that invents (and enforces) its own rules.

**Comment by fivaldi** [comment]
No problem. Let's close this without merging. We can discuss some time...

**Comment by encukou** [comment]
No, most of the changes are good! I'm just wary of adding a mandatory check to CI.

I'll merge the formatting changes (when I get to it). I'll look at the failing test, too.

**Comment by encukou** [comment]
Added without the CI check.

**Pull request closed without merging**